SECURITY.md

```diff
@@ -53,6 +53,86 @@ Here is the security disclosure policy for Node.js
   the release process above to ensure that the disclosure is handled in a
   consistent manner.
 
+## What does Node.js trust/not trust
+
+Being able to cause a negative outcome in a way that requires control of the
+elements that are trusted by Node.js is not considered a vulnerability in
+Node.js.
+
+Being able to cause the following through control of the elements that Node.js
+does not trust is considered a vulnerability:
+
+* Disclosure or loss of integrity or confidentiality of data protected through
+  the correct use of Node.js APIs.
+* The unavailability of the runtime, including the unbounded degradation of its
+  performance.
+
+If Node.js loads configuration files or runs code by default (without a
+specific request from the user), and this is not documented, it is considered a
+vulnerability.
+Vulnerabilities related to this case may be fixed by a documentation update.
+
+**Node.js does NOT trust**
+
+1. The data from network connections that are created through the use of Node.js
+   APIs and which is transformed/validated by Node.js before being passed to the
+   application. This includes:
+   * HTTP APIs (all flavors) client and server APIs.
+   * DNS APIs.
+2. Consumers of data protected through the use of Node.js APIs (for example
+   people who have access to data encrypted through the Node.js crypto apis).
+3. The file content or other IO that is opened for reading or writing by the use
+   of Node.js APIs (ex: stdin, stdout, stderr). In other words, Node.js doesn’t
+   interpret buffers unless explicitly set by the user.
```

Comment:
```suggestion
3. The file content or other IO that is opened for reading or writing by the use
   of Node.js APIs (ex: stdin, stdout, stderr). In other words, if the content passing
   through Node.js to/from the application can trigger actions other than
   those documented for the APIs, the content is not trusted avoid causing
   those actions.
```

Reply:
@RafaelGSS made a suggetion for this one. Still finding it hard to clearly explain.

Reply:
Let's use `I/O` instead of `IO`. https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/input-output